lms/static/scripts/frontend_apps/components/URLPicker.tsx

```diff
@@ -74,9 +83,10 @@ export default function URLPicker({
         </Button>,
         <Button
           data-testid="submit-button"
+          form={formId}
```

Comment:
TIL https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#form

I remember trying to make this button a submit one, and having to discard it because it was technically not "in" the form. It was this easy 😅